flink-clients/src/main/java/org/apache/flink/client/ClientUtils.java

```diff
@@ -124,6 +47,7 @@ public static void checkJarFile(URL jar) throws IOException {
 			throw new IOException("JAR file can't be read '" + jarFile.getAbsolutePath() + '\'');
 		}
 
+		//noinspection EmptyTryBlock
```

Comment:
Thanks for your information. I think it is not a warning for java compiler. Besides, we over 100 usages of `//noinspection`. I'm ok to revert this line fair enough but it might be worth a community discussion :-)

Reply:
Yes, having a discussion about this is a good point.